protocol/agents.json

```diff
@@ -89,6 +89,14 @@
             ],
             "type": "sdk"
         },
+        {
+            "identifier": "jre",
+            "versioned": true,
+            "emitters": [
+                "ably-java"
+            ],
+            "type": "sdk"
```

Comment:
Since JRE is a runtime environment, this should be `"type": "runtime"`

Reply:
Sorry, copy-paste mistake :wink: Thanks for catching it :bow: https://github.com/ably/ably-common/pull/95/commits/34dd7092ae011c2319bc6d90577f4763f1071a26